### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by jlnostr**
The app can now be translated into all the different languages. I've made some line comments on the store description files, maybe one of you guys does have an idea for the content :)

(ping @tcitworld @nicosomb @j0k3r)

### CODE REVIEW

**Comment by tcitworld** [comment]
@nicosomb can advertise on social networks for translations. ;-)

File: store/App descriptions/fr_FR.md [hunk 1]
```diff
@@ -6,12 +6,23 @@ Gr�ce � cette application, vous ne raterez plus aucun contenu. Sauvegardez q
 Il y a deux mani�res : vous installez wallabag sur votre serveur web ou alors vous vous cr�ez un compte gratuit sur Framabag (nous g�rons l�installation et la mise � jour pour vous).
```
Comment (j0k3r) [inline store/App descriptions/fr_FR.md]: Does the broken character (the encoded one) are _normal_ ?

Comment (jlnostr) [inline store/App descriptions/fr_FR.md]: They shouldn't be, but they are. Maybe someone could update the file :wink:

**Comment by SweetLion** [comment]
Just saw your Tweet. What do you think of Transifex & Co?

**Comment by jlnostr** [comment]
Thought about it. You can find the project on [OneSkyApp](http://www.oneskyapp.com/) now, too :)
Should I invite you for a specific language, @SweetLion?

**Comment by SweetLion** [comment]
Thanks, I would like to help you with the German translation.

**Comment by jlnostr** [comment]
Not necessary, I'm a native german speaker. Aber trotzdem danke :D

**Pull request merged**